tests/tests_nn/test_unet_2d.py

```diff
@@ -58,4 +65,12 @@ def test_unet_2d(shape, num_filters, num_pool_layers, skip, normalized):
 
     out = model(data, sens)
 
+    if openvino_available:
+        ov_model = OpenVINOModel(model)
+        # ov_out = ov_model(masked_kspace=data, sensitivity_map=sens)
+        ov_out = ov_model(masked_kspace=data)
+
+        assert torch.max(torch.abs(out[0][-1] - ov_out[0][-1])) < 1e-5
+        assert torch.max(torch.abs(out[1] - ov_out[1])) < 1e-5
```

Comment:
Do we know a source of second output? According to CI, it's something related to gradients:
```
assert tensor(5.5534, grad_fn=<MaxBackward1>) < 1e-05
```
Can you try to keep only 0th check?